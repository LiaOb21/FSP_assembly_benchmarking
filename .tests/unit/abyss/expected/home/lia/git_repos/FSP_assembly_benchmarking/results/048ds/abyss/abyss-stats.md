n    |n:500  |L50  |min  |N75  |N50  |N25  |E-size  |max  |sum  |name
---  |---    |---  |---  |---  |---  |---  |---     |---  |---  |---
139  |0      |0    |0    |0    |0    |0    |0       |0    |0    |abyss-unitigs.fa
139  |0      |0    |0    |0    |0    |0    |0       |0    |0    |abyss-contigs.fa
139  |0      |0    |0    |0    |0    |0    |0       |0    |0    |abyss-scaffolds.fa
