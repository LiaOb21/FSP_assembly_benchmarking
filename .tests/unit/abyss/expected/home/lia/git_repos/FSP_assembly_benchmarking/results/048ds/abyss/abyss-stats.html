<table>
<colgroup>
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
<col style="width: 9%" />
</colgroup>
<thead>
<tr class="header">
<th>n</th>
<th>n:500</th>
<th>L50</th>
<th>min</th>
<th>N75</th>
<th>N50</th>
<th>N25</th>
<th>E-size</th>
<th>max</th>
<th>sum</th>
<th>name</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>139</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>abyss-unitigs.fa</td>
</tr>
<tr class="even">
<td>139</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>abyss-contigs.fa</td>
</tr>
<tr class="odd">
<td>139</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>abyss-scaffolds.fa</td>
</tr>
</tbody>
</table>
